common-money.lic

```diff
@@ -129,4 +129,23 @@ module DRCM
     minimize_coins(keep_copper).each { |amount| withdraw_exact_amount?(amount, settings) }
     DRC.bput('check balance', 'your current balance is')
   end
+
+  def town_currency(town)
+    get_data('town')[town]['currency']
+  end
+
+  def get_money_from_specific_bank(amount_as_string, town)
```

Comment:
❓ If `get_money_from_specific_bank` is going to return a bool, could you add `?` to the end of the name to indicate it is a predicate?

Reply:
Sure thing, I pilfered from `get_money_from_bank` which also is a predicate method, do you want a ? there too? (Though it will break stuff for users.)

Reply:
@rpherbig

Reply:
I would not change the method that is already being used

Reply:
ok